app/views/portfolios/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= csrf_meta_tags %>
+
+<h2>User Details</h2>
+
+ <h3><%= current_user.email%></h3>
+
+<%= link_to 'Sign out', destroy_user_session_path, method: :delete %>
```

Comment:
add new line at the end of file

Reply:
Will do, maam. May I ask why it is a good practice to add new lines at the end of a file?

Reply:
To avoid Github's warning sign
![image](https://user-images.githubusercontent.com/9253881/134007407-f31de1b4-2d99-4b1e-896d-49d9de3b254f.png)